Add 11.30 release notes (#458)
mondoohq/docs: releases/2024-11-12-mondoo-11.30-is-out.md

---
slug: mondoo-11.30-is-out/
title: Mondoo 11.30 is out!
description: Announcing the 11.30 release of Mondoo with notes for cases, expanded Windows snapshot scanning, updated policies, and more!
authors: [tim]
image: /img/releases/2024-11-12-mondoo-11.30-is-out/notes.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.30 is out! This release includes notes for cases, expanded Windows snapshot scanning, updated policies, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Notes for cases

Provide the context that downstream teams need to quickly fix security issues in your spaces with notes for cases, which automatically sync to your ticketing integrations.

![Notes](/img/releases/2024-11-12-mondoo-11.30-is-out/notes.png)

### Package data in Windows snapshot scans

> Expose package vulnerabilities on Windows cloud hosts without deploying agents with Windows package discovery for cloud snapshot scans.

Reviewer: ```suggestion
You can now expose package vulnerabilities on Windows cloud hosts without deploying agents. Windows  cloud snapshot scans now include package discovery.
```

### Parse fstab entries with MQL

Dive head first into Linux fstab entries with new MQL resources for fstab data.

```coffee
fstab.entries: [
  0: {
    fsck: 1
    options: "defaults,noatime"
    device: "UUID=7c4e7e0e-ce36-42f9-b456-16f78f3a1eb1"
    fstype: "xfs"
    mountpoint: "/"
    dump: 1
  }
  1: {
    fsck: 2
    options: "defaults,noatime,uid=0,gid=0,umask=0077,shortname=winnt,x-systemd.automount"
    device: "UUID=C155-24D2"
    fstype: "vfat"
    mountpoint: "/boot/efi"
    dump: 0
  }
]
```

### Discover VMware distributed virtual switches and port groups

> Discover and secure VMware distributed virtual switches and portgroups with new and expanded MQL resources for VMware.

Reviewer: ```suggestion
Mondoo's new and expanded MQL resources for VMware let you discover and secure VMware distributed virtual switches and port groups.
```
Just so it isn't identical to the heading.
